src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -47,7 +47,13 @@
     public function load(/*...$paths*/)
```

Comment:
if we want to enforce one arg, what about doing it here `load($path/*, ...$paths*/)`?